src/osemgrep/cli_ci/Ci_subcommand.ml

```diff
@@ -125,15 +125,16 @@ let run (conf : Ci_CLI.conf) : Exit_code.t =
             Fmt.(styled `Bold string)
             (Option.value ~default:"unknown" metadata.Project_metadata.on));
       (* TODO: fix_head_if_github_action(metadata) *)
-      let _scan_id, _rules_and_origins =
+      let r =
```

Comment:
maybe a type annotation there would help understand what 'r' contains. I guess it's a result,
but what is the Ok xx value in it.
Maybe a comment also would help.